src/network/networkd-link.c

```diff
@@ -469,6 +469,10 @@ static int link_new(Manager *manager, sd_netlink_message *message, Link **ret) {
                         return -ENOMEM;
         }
 
+        r = sd_netlink_message_read_u32(message, IFLA_MASTER, (uint32_t *)&link->master);
+        if (r < 0)
+                log_link_debug_errno(link, r, "new device has no master, continuing without");
```

Comment:
Please capitalize the log message: "New device has...". And please append ": %m".